cardano-cli/test/cardano-cli-golden/Test/Golden/Babbage/Governance/AnswerPoll.hs

```diff
@@ -30,43 +30,43 @@ hprop_golden_shelleyGovernanceAnswerPollNeg1Invalid = propertyOnce . H.moduleWor
 
   either (const H.success) (const H.failure) result
 
-hprop_golden_shelleyGovernanceAnswerPoll0 :: Property
-hprop_golden_shelleyGovernanceAnswerPoll0 = propertyOnce . H.moduleWorkspace "governance-answer-poll" $ \tempDir -> do
-  pollFile <- noteInputFile "test/cardano-cli-golden/files/golden/shelley/governance/polls/basic.json"
-  expectedAnswerFile <- noteInputFile "test/cardano-cli-golden/files/golden/shelley/governance/polls/basic.answer.0.json"
+hprop_golden_babbageGovernanceAnswerPoll0 :: Property
+hprop_golden_babbageGovernanceAnswerPoll0 = propertyOnce . H.moduleWorkspace "governance-answer-poll" $ \tempDir -> do
+  pollFile <- noteInputFile "test/cardano-cli-golden/files/golden/babbage/governance/polls/basic.json"
+  expectedAnswerFile <- noteInputFile "test/cardano-cli-golden/files/golden/babbage/governance/polls/basic.answer.0.json"
```

Comment:
This is a golden file, so the variable name should say `goldenAnswerFile`.

Golden files are not input files, so use `note` instead of `noteInputFile`.

Reply:
Changed the names :+1: Thanks for the suggestions.